src/registrar/config/urls.py

```diff
@@ -149,6 +149,10 @@
     ),
 ]
 
+# Djangooidc strips out context data from that context, so we define a custom error
+# view through this method.
+handler500 = "registrar.views.utility.error_views.custom_500_error_view"
```

Comment:
Suggestion: The comment can be a bit clearer. Do we know why and how Djangooidc strips out the context? The custom error view preserve the context as it takes it in a an optional param and passes it along to the render func?